README.md

```diff
@@ -1,2 +1,25 @@
-# opa
-A library of policy algorithms deployed as an agent to policy-enable any product or service
+# OPA: Open Policy Agent
+
+OPA is an open source project which can help policy enable any application.
+
+## What is Policy?
+
+Policy defines expected behavior in response to specific events within an application. The implementation of policy can vary greatly within and across applications, e.g, it may be maintained manually by administrators, documented on wiki pages, hardcoded in applications, exposed through configuration, etc. Often, these policies are not part of the application's core business logic.
+
+There are many examples of functionality within applications which can benefit from rich policy control, e.g, API authorization, VM and container placement, auto-scaling, auto-healing, etc.
+
+## What is Policy Enabling?
+
+Policy enabling an application decouples the policy implementation from the business logic so that administrators can define policy without changing the application while still keeping up with the size, complexity, and dynamic nature of modern applications.
+
```

Comment:
Could we add an explicit example of how someone might policy-enable their app?  Maybe take the API use case and compare doing it with an RBAC system and with OPA.